src/i18n/en.json

```diff
@@ -867,6 +906,18 @@
         "5": "Simple Action (*5)"
       }
     },
+    "RQGSystem": {
+      "rqid": "Rune Quest System ID",
+      "QuickRqid": "Create Rune Quest System ID (rqid) based on current item. Item should be localized in English.",
+      "rqidpriority": "System ID Priority",
+      "rqidlang": "System ID Language",
```

Comment:
I went with "This Item Localized to"

Reply:
My inital feeling was "Item language", but I backed out of that in case we extend this to other document types. But lets leave it like this and review that when we get there!